### PR TITLE
add pruner unit test

### PR DESCRIPTION
**Pull request by tanglang96**
add l1filter_pruner and slim_pruner unit test

### CODE REVIEW

File: docs/en_US/Compressor/SlimPruner.md [hunk 1]
```diff
@@ -34,6 +34,6 @@ We implemented one of the experiments in ['Learning Efficient Convolutional Netw
 | Model         | Error(paper/ours) | Parameters | Pruned    |
 | ------------- | ----------------- | ---------- | --------- |
 | VGGNet        | 6.34/6.40     | 20.04M   |           |
-| Pruned-VGGNet | 6.20/6.39     | 2.03M    | 88.5% |
+| Pruned-VGGNet | 6.20/6.26     | 2.03M    | 88.5% |
```
Comment (xuehui1991) [inline docs/en_US/Compressor/SlimPruner.md]: could you provide some settings in those experiments? like hyper-parameters.

File: examples/model_compress/slim_pruner_torch_vgg19.py [hunk 2]
```diff
@@ -169,7 +169,7 @@ def main():
     new_model.to(device)
     new_model.load_state_dict(torch.load('pruned_vgg19_cifar10.pth'))
     test(new_model, device, test_loader)
-    # top1 = 93.61%
+    # top1 = 93.74%
```
Comment (xuehui1991) [inline examples/model_compress/slim_pruner_torch_vgg19.py]: does this value will change by some random seed? If will, we should not assume the top1 value is a specific value.
